src/main/java/org/mule/extension/sftp/internal/SftpOperations.java

```diff
@@ -250,7 +249,7 @@ public void copy(@Config FileConnectorConfig config, @Connection FileSystem file
    * @throws IllegalArgumentException if an illegal combination of arguments is supplied
    */
   @Summary("Moves a file")
-  @Throws(FileCopyErrorTypeProvider.class)
+  @Throws(SftpFileCopyErrorTypeProvider.class)
   public void move(@Config FileConnectorConfig config, @Connection FileSystem fileSystem,
```

Comment:
isn't SftpFile**Move**ErrorTypeProvider worth doing?

Reply:
Since we have the same error codes for both operations don't think it is worth create a duplicate class just to have better naming.
Maybe there is a better naming for both of them. What ideas do you have?